markdown/tuto-020-shell-tools.md

```diff
@@ -245,16 +252,26 @@ You can run a command on all selected nodes with
 
 this time of course, you cannot specify another set of nodes than the selection.
 
+If you want to directly access a Bash prompt **inside a Docker container** please choose the username `container`, like this :
+
+    ssh container@fit25
```

Comment:
so, I am confused here

just above there is a script that uses 'port=2222' to join the container - IIUC
but here you say I enter the container using the `container` login but o port 22 - which one is it ? or is it both ?